pipeline-scripts/commonlib.groovy

```diff
@@ -138,7 +138,7 @@ def describeJob(name, description) {
         job.setDisplayName(name)
     }
     if (description) {
-        job.setDescription(description)
+        job.setDescription(description.readLines().join('<br>\n'))
```

Comment:
oh man... way to bury the lead github! heh... that'll do for now, we'll probably want an option later to not do this :)